package.json

```diff
@@ -43,8 +43,10 @@
   },
   "homepage": "https://github.com/mostjs/dom-event#readme",
   "devDependencies": {
+    "@briancavalier/assert": "^3.4.0",
+    "@most/core": "^0.13.0",
```

Comment:
What do you think about also switching to our new [standard lint setup](https://github.com/mostjs/core/blob/master/packages/core/package.json#L16) in this PR?